terraform/environments/digital-prison-reporting/modules/rds/aws-aurora/main.tf

```diff
@@ -0,0 +1,458 @@
+data "aws_partition" "current" {}
+
+locals {
+  create = var.create && var.putin_khuylo
```

Comment:
Not sure we should leave this second var in here...

Reply:
updated